docs/reference/nodejs/examples.md

```diff
@@ -510,296 +511,289 @@ Takes an http request and creates an http signature using a private key.
+| `ledger`     | a string identifying both the network and ledger-id      |
 
-### Returns {#returns-13}
+### Returns {#ledgerInfo-returns}
 
 Returns a map with information about the requested ledger.
 
-### Code Example {#code-example-12}
+### Code Example {#ledgerInfo-example}
 
 ```javascript
-  const flureenjs = require('@fluree/flureenjs');
-  const flureeServerUrl = "http://localhost:8090";
-  const ledger = "test/chat";
-
-  let flureeDbConn = flureenjs.connect(flureeServerUrl);
-  :
-  :
-  flureenjs.ledger_info(flureeDbConn, ledger)
-            .then (results => {
-                console.log(results);
-            })
-            .catch(error => {
-                console.log(error);
-            });
-:
-:
+const flureenjs = require("@fluree/flureenjs");
+const flureeServerUrl = "http://localhost:8090";
+const ledger = "test/chat";
+
+let flureeDbConn = flureenjs.connect(flureeServerUrl);
+
+flureenjs
+  .ledgerInfo(flureeDbConn, ledger)
+  .then((results) => {
+    console.log(results);
+  })
+  .catch((error) => {
+    console.log(error);
+  });
 ```
 
-## **ledger_list** {#ledger_list}
+## **`ledgerList`** {#ledgerList}
 
 Returns a promise that eventually contains a list of ledgers that the connected server is currently serving.
 
-### Parameter(s) {#parameters-13}
+### Parameter(s) {#ledgerList-parameters}
 
-| Name         | Value                                                                   |
-| ------------ | ----------------------------------------------------------------------- |
-| `connection` | a connection object created using the `connect` or `connect_p` function |
+| Name         | Value                                                    |
+| ------------ | -------------------------------------------------------- |
+| `connection` | a connection object created using the `connect` function |
 
-### Returns {#returns-14}
+### Returns {#ledgerList-returns}
 
 Returns a list of ledgers serviced by the connected Fluree instance.
 
-### Code Example {#code-example-13}
+### Code Example {#ledgerList-example}
 
 ```javascript
-  const flureenjs = require('@fluree/flureenjs');
-  const flureeServerUrl = "http://localhost:8090";
-  const ledger = "test/chat";
-
-  let flureeDbConn = flureenjs.connect(flureeServerUrl);
-  :
-  :
-  flureenjs.ledger_list(flureeDbConn)
-            .then (results => {
-                console.log(results);
-            })
-            .catch(error => {
-                console.log(error);
-            });
-:
-:
+const flureenjs = require("@fluree/flureenjs");
+const flureeServerUrl = "http://localhost:8090";
+const ledger = "test/chat";
+
+let flureeDbConn = flureenjs.connect(flureeServerUrl);
+
+flureenjs
+  .ledgerList(flureeDbConn)
+  .then((results) => {
+    console.log(results);
+  })
+  .catch((error) => {
+    console.log(error);
+  });
 ```
 
-## **ledger_stats** {#ledger_stats}
+## **`ledgerStats`** {#ledgerStats}
```

Comment:
`ledgerStats` is deprecated, `ledgerInfo` is what people should use going forward.

"Returns promise with ledger's status as a map, including index, indexes, block, and status.
  If ledger doesn't exist, will return an empty map."